homeassistant/components/shelly/__init__.py

```diff
@@ -249,6 +255,7 @@ async def _async_setup_rpc_entry(hass: HomeAssistant, entry: ConfigEntry) -> boo
         entry.data.get(CONF_USERNAME),
         entry.data.get(CONF_PASSWORD),
         device_mac=entry.unique_id,
+        port=entry.data.get(CONF_PORT, DEFAULT_HTTP_PORT),
```

Comment:
`data.get(CONF_PORT, DEFAULT_HTTP_PORT)` this repeats 3 times in this PR, can we make it a method that receives the `data` dict and return the port?

`get_http_port`

Reply:
This is still not addressed

Reply:
Done